src/main/java/com/example/demo/translation/T.java

```diff
@@ -0,0 +1,106 @@
+package com.example.demo.translation;
+
+
+import com.vaadin.flow.server.VaadinSession;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Translator
+ */
```

Comment:
I doubt this is a good place for this addition. It sounds more like an add-on and not anything that every developer wants in this way because it's highly subjective.

Reply:
I'm not sure that you can do it as an addon. You need to be able to modify the T class directly.

Reply:
In the end your implementation is just a map with static strings as key in a class as wrapper. So the whole translation part can be an add-on and you just pass an argument to your add-on; like with any other translation that is key-value based. 

I really don't see the benefit from this vs 

getTranslation(MyStaticClass.HELLO_WORLD)

Reply:
Well you see the difference right there, do you want to write `getTranslation(MyStaticClass.HELLO_WORLD)` every time or `T.HELLO_WORLD+""`